pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotTableRestletResource.java

```diff
@@ -367,6 +432,16 @@ public String checkTableConfig(String tableConfigStr) {
       throw new ControllerApplicationException(LOGGER, "Invalid table config", Response.Status.BAD_REQUEST, e);
     }
   }
+  
+  private List<String> getExistingTableNamesWithType(String tableName, @Nullable TableType tableType) {
```

Comment:
tableType doesn't need annotation Nullable - you're checking for not-null before this call